src/code/templates/fv-egg-game.js

```diff
@@ -1233,6 +1252,7 @@ export default class FVEggGame extends Component {
   }
 
   componentWillUnmount() {
+    this.props.onResetGametes();
```

Comment:
Was seeing some state preserved between level navigation until I reset the gametes before leaving a level